Demo/README.md

```diff
@@ -5,8 +5,8 @@
 For this demo we show the capabilities of the LSOracle tool on a benchmark from the OpenPiton Design Benchmark (OPDB) suite. Optimization of this file requires only 3 commands: read_aig, partitioning, and optimization.
 
 ```{r, engine='bash', count_lines}
-./lstools                                                #run the tool
+../build/core/lsoracle                                   #run the tool
```

Comment:
i think you only want one dot prefix

Reply:
or no dots, since it's not in the working directory

Reply:
the two dot prefix just navigates from the working directory to the directory where the tool is in, and the working directory is just assumed to be demo, because that's where this file is.